### ISSUE TITLE
flow.json accounts formatting not consistent for various actions

### ISSUE THREAD

**Issue by polluterofminds** [onflow/flow-cli#112]
### Problem

The format of the `accounts` object in the `flow.json` config needs to be different when starting the emulator and deploying a project than it needs to be when executing transactions. For example, when starting the emulator and deploying a project, the accounts object needs to look something like this: 

```json
"accounts": {
	"emulator-account": {
		"address": "f8a6e0586a0a20c7",
		"keys": "e5ca2b0946350223b0555206144fe4d74e65cbd58b0933c5232ce195b9058cdd"
	}
},
```

However, when executing transactions, the accounts object needs to look like this: 

```json
"accounts": {
	"emulator-account": {
		"address": "f8a6e0586a0a20c7",
		"privateKey": "e5ca2b0946350223b0555206144fe4d74e65cbd58b0933c5232ce195b9058cdd",
		"sigAlgorithm": "ECDSA_P256",
		"hashAlgorithm": "SHA3_256",
		"chain": "flow-emulator"
	}
},
```

It is VERY possible that I am misunderstanding the way the CLI is designed to work, so please let me know if that's the case.

**Comment by polluterofminds** [comment]
Maybe this is fixed in the newest version of the CLI, but I'm not seeing the problem anymore.

**Comment by sideninja** [comment]
We've made a quickfix for this in the latest version. For future reference, the new configuration allows both formats, first one is meant to be a simple format and the second one is an advanced format where you can specify all the values, however, if you use default values you can just use the simple format.

**Comment by sideninja** [comment]
Solved in #104
